pkgs/servers/x11/xorg/default.nix

```diff
@@ -2526,6 +2526,7 @@ let
     nativeBuildInputs = [ pkgconfig ];
     buildInputs = [ libX11 xproto ];
     meta.platforms = stdenv.lib.platforms.unix;
+    meta.license = stdenv.lib.licenses.mit;
```

Comment:
This one is generated. You have to edit overrides.nix here: https://github.com/NixOS/nixpkgs/blob/274f091b9531bb1efc446c3f05be91d5ebca372f/pkgs/servers/x11/xorg/overrides.nix#L37

Reply:
Thank you. Should be fixed now.